### PR TITLE
Bump versions (autoprefixer and postcss)

### PR DESCRIPTION
**Pull request by BowlingX**
Hey,
I bumped the versions of autoprefixer and postcss, added more whitelist options and removed deprecated options.

David

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var loaderUtils = require('loader-utils');
 var autoprefixerCore = require('autoprefixer-core');
 var postcss = require('postcss');
 var path = require('path');
+var safe = require('postcss-safe-parser');
```
Comment (passy) [inline index.js]: This doesn't seem to be used?

Comment (BowlingX) [inline index.js]: It's used on line 25

Comment (passy) [inline index.js]: Ohh, sorry I missed that one.

**Comment by passy** [comment]
@BowlingX Thanks for that! :) Would you mind rebasing your patch? I made some cleanups in the meantime.

**Comment by BowlingX** [comment]
I just merged your changes with my branch :)

**Comment by passy** [comment]
I rebased and squashed the changes again, but they're live now. Thanks again! 🎉

**Pull request closed without merging**